src/services/userService.ts

```diff
@@ -245,11 +258,28 @@ export const changeEmail = async (email: string, ott: string) => {
     );
 };
 
-export const getUserDetails = async (): Promise<UserDetails> => {
+export const getUserDetailsV2 = async (): Promise<UserDetails> => {
     const token = getToken();
 
-    const resp = await HTTPService.get(`${ENDPOINT}/users/details`, null, {
-        'X-Auth-Token': token,
-    });
-    return resp.data['details'];
+    const resp = await HTTPService.get(
+        `${ENDPOINT}/users/details/v2?memoryCount=true`,
+        null,
+        {
+            'X-Auth-Token': token,
+        }
+    );
+    return resp.data;
+};
+
+export const getFamilyPortalRedirectURL = async () => {
+    try {
+        const jwtToken = await getFamiliesToken();
+        const isFamilyCreated = false;
```

Comment:
why are we setting it to false always , shouldn't we check the familyData to see it user has a active family or not ?